lib/erb_lint/runner_config.rb

```diff
@@ -45,24 +45,30 @@ def merge!(other_config)
     end
 
     class << self
-      def default
+      def default(default_enabled: true)
```

Comment:
nit: I'd make the default `nil` now. That way the default value is only in one place (the line below this).

I forgot to mention, the one nice thing about doing it this way is that we can change the default value for when the option is not specified. Not that we _should_ do that, but now it's easy to do so, and in the right place 🙂